store/tikv/gcworker/gc_worker_test.go

```diff
@@ -43,18 +38,19 @@ type testGCWorkerSuite struct {
 	store    tikv.Storage
 	oracle   *mockoracle.MockOracle
 	gcWorker *GCWorker
+	dom      *domain.Domain
 }
 
 var _ = Suite(&testGCWorkerSuite{})
 
 func (s *testGCWorkerSuite) SetUpTest(c *C) {
 	tikv.NewGCHandlerFunc = NewGCWorker
-	var err error
-	s.store, err = mockstore.NewTestTiKVStorage(*withTiKV, *pdAddrs)
+	store, err := mockstore.NewMockTikvStore()
```

Comment:
Why remove `withTiKV` case?

Reply:
Run `store/tikv` package tests using the `with-tikv` flag will use the TiKV storage, tests have to run one by one. (since there is only one storage instance)
If `store/tikv/gcworker` package test runs concurrency with other tests, TiKV storage may be disturbed by it.
So I  choose not to run gcworker test under TiKV.